zeppelin-zengine/src/test/java/org/apache/zeppelin/util/UtilsForTests.java

```diff
@@ -110,10 +115,10 @@ public static void setEnv(String k, String v) {
 	            }
 	        }
 	      } catch (Exception e2) {
-	        e2.printStackTrace();
+					LOGGER.error(e2.toString());
```

Comment:
I have fixed this entire file, previously it had a lot of tab characters, now made it to all spaces ( tab size :2, Indent :2, Continuation indent :2)